### PR TITLE
Convert VR tests to single completion handler

### PR DESCRIPTION
**Pull request by mkistler**
Initial pass at changing VR tests to single completion handler.

### CODE REVIEW

File: Tests/VisualRecognitionV3Tests/VisualRecognitionTests.swift [hunk 1]
```diff
@@ -173,8 +175,16 @@ class VisualRecognitionTests: XCTestCase {
...
-        visualRecognition.createClassifier(name: name, positiveExamples: classes, failure: failWithError) {
-            classifier in
+        visualRecognition.createClassifier(name: name, positiveExamples: classes) {
+            response, error in
+            if let error = error {
+                XCTFail("Unexpected error response from service: \(error)")
+                return
+            }
+            guard let classifier = response?.result else {
+                XCTFail("Unexpected response value")
```
Comment (AnthonyOliveri) [inline Tests/VisualRecognitionV3Tests/VisualRecognitionTests.swift]: I think we should reword this to `Missing response value`.

**Comment by AnthonyOliveri** [comment]
I ran the tests, and got a couple of failed tests (due to timeouts). I ran them again and they all succeeded.

File: Tests/VisualRecognitionV3Tests/VisualRecognitionTests.swift (hunk 1 above)
Comment (mkistler) [reply Tests/VisualRecognitionV3Tests/VisualRecognitionTests.swift]: Agreed.  I'll make that change.

**Comment by mkistler** [comment]
I think this is ready to go now.  I removed the do-not-merge label.

**Comment by AnthonyOliveri** [comment]
After reading those `XCTFail` messages, I think they would be clearer if written like this:

`Missing result from response`
`Received an unexpected error: \(error)`
`Expected to receive an error`

**Comment by mkistler** [comment]
I like the first two.

For the last, how about `Expected error not received`?

**Comment by AnthonyOliveri** [comment]
That's fine with me. To keep things from getting too messy, let's keep the error messages as they are now. Once all of the tests have been updated and PRs are merged, I'll make a new PR to update all of the messages.
